.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+- [ ] Commits are squashed in a cohesive manner and have meaningful messages (see [how](https://cbea.ms/git-commit))
+- [ ] The robocop analyser doesn't detect any issues in the test suites modified by this PR (see [check-code-style](https://github.com/red-hat-data-services/ods-ci/blob/master/docs/check-code-style.md))
+- [ ] Code is formatted following the _[ODS-CI RobotFramework Style Guide](https://docs.google.com/document/d/11ZJOPI1uq-0Wl6a2V8fkAv_TQhfzp9t_IjXAheaJxmQ/edit?usp=sharing)_
+
+For new automated tests:
+- [ ] Test case is authored in Polarion
+- [ ]  _Automation = Automated_ in Polarion
+- [ ] _Automated = Yes_ in Jira
+- [ ] Test case has tier information and Polarion ID in its Tags (more info [here](https://docs.google.com/document/d/11ZJOPI1uq-0Wl6a2V8fkAv_TQhfzp9t_IjXAheaJxmQ/edit#heading=h.s819p3c5ud7p))
```

Comment:
If/When we start to add automation for features that are in upstream ODH and won't have a polarion testcase, will not checking these boxes break any CI automation tied to this template?

Reply:
It is possible that  we will set up GiHub actions to verify that robocop analyser doesn't detect any issues, but there is no automation planned about the items you mentioned (it's more like a reminder for us).

If you contribute with features present in ODH but not in RHODS we'll need a tag to indicate this so we don't run those tests.  We'll give a thought and document it in the style guide

We could do this:

- [ ] Commits are squashed in a cohesive manner and have meaningful messages (see [how](https://cbea.ms/git-commit))
- [ ] The robocop analyser doesn't detect any issues in the test suites modified by this PR (see [check-code-style](https://github.com/red-hat-data-services/ods-ci/blob/master/docs/check-code-style.md))
- [ ] Code is formatted following the _[ODS-CI RobotFramework Style Guide](https://docs.google.com/document/d/11ZJOPI1uq-0Wl6a2V8fkAv_TQhfzp9t_IjXAheaJxmQ/edit?usp=sharing)_

For new automated tests for RHODS (not ODH):
- [ ] Test case is authored in Polarion
- [ ]  _Automation = Automated_ in Polarion
- [ ] _Automated = Yes_ in Jira
- [ ] Test case has tier information and Polarion ID in its Tags (more info [here](https://docs.google.com/document/d/11ZJOPI1uq-0Wl6a2V8fkAv_TQhfzp9t_IjXAheaJxmQ/edit#heading=h.s819p3c5ud7p))